### PR TITLE
Jdwykiki/story

### PR DESCRIPTION
**Pull request by Jdwykiki**
I looked at your app and thought I could help!

### CODE REVIEW

**Comment by cmckee-dev** [comment]
This file is a .rdoc, can we please change the extention!

**Comment by jasonamclean** [comment]
'extention' is spelled 'extension'

**Comment by collinthedavis** [comment]
Sounds like a good movie.

**Comment by collinthedavis** [comment]
Sounds like it would make a good movie.

File: README.rdoc [hunk 1]
```diff
@@ -1,28 +1,7 @@
-== README
+The arabian sea was flawless, and clear, dolaphins and string rays could be visiable swimming along side the vessel. The crew was small, 15 or so. The ranks were filled by local fishermen, scientist, and archeologists they were entering the cliffy cove, a 10 foot braker smashed the rear motor, instantaniously striping the moter of its four blades and in the process pulling the ligament of the hired help. The dig site was a mile inland, and there was no possibility of staying the night.
```
Comment (jasonamclean) [inline README.rdoc]: string rays  = sting rays
dolaphins = dolphin
visiable = visible
braker = breaker
striping = stripping
instantaniously = instantaneously

GOOD LUCK!

File: README.rdoc [hunk 2]
```diff
@@ -1,28 +1,7 @@
-== README
+The arabian sea was flawless, and clear, dolaphins and string rays could be visiable swimming along side the vessel. The crew was small, 15 or so. The ranks were filled by local fishermen, scientist, and archeologists they were entering the cliffy cove, a 10 foot braker smashed the rear motor, instantaniously striping the moter of its four blades and in the process pulling the ligament of the hired help. The dig site was a mile inland, and there was no possibility of staying the night. 
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+They healed the man of his wounds and helped him into the base camp, and continued to the the ancient temple site, with in hours they had found an ancient israelite alter decorated with the 12 cows that symbolize the hebrew nation, the temple, exotically located nations away from its hailing land near palestine. 
 
-Things you may want to cover:
+Hours later a small group preformed a scarament that had not been preformed in this location for 2600 years.
```
Comment (silvercoulson) [inline README.rdoc]: Scarament

**Comment by Anthony7147** [comment]
Great story

**Comment by jasonamclean** [comment]
Here's Justin's story :)

+The arabian sea was flawless, and clear, dolaphins and string rays could be visiable swimming along side the vessel. The crew was small, 15 or so. The ranks were filled by local fishermen, scientist, and archeologists they were entering the cliffy cove, a 10 foot braker smashed the rear motor, instantaniously striping the moter of its four blades and in the process pulling the ligament of the hired help. The dig site was a mile inland, and there was no possibility of staying the night.

**Pull request closed without merging**